tests/test_dtr.py

```diff
@@ -24,6 +24,7 @@
 import numpy as np
 import pytest
 
+from mdtraj.core.trajectory import _parse_topology
```

Comment:
I understand what you're doing with `_parse_topology` but I'm curious if important an internal method to `Trajectory` objects is the best way we should be loading and parsing topologies for testing. Is there a better way we can load and pass the topologies to `read_as_traj`? I know that in `test_hdf5` we use `md.load_pdb().topology`